src/BaGet/Program.cs

```diff
@@ -4,63 +4,79 @@
 using McMaster.Extensions.CommandLineUtils;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.DependencyInjection;
+using System.IO;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Hosting;
 
 namespace BaGet
 {
-    public class Program
+    namespace BaGet
```

Comment:
Do we need to redeclare the namespace?

Reply:
No, I guess, this was some strange kind of copy paste error...